fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -319,13 +336,25 @@ protected RecordMetaData buildMetaData(RecordMetaDataProto.MetaData metaDataProt
      */
     @Nonnull
     public CompletableFuture<Void> preloadMetaData(@Nullable RecordMetaDataProvider metaDataProvider) {
+        return preloadMetaData(metaDataProvider, null);
+    }
+
+    /**
+     * Prepare a meta-data store for use by loading any existing {@link RecordMetaData} or storing an initial
+     * seed from the given provider.
```

Comment:
Same comment about linking to `setLocalFileDescriptor` or something along those lines.